docs/gathered-data.md

```diff
@@ -43,13 +43,17 @@ The CRD sizes above are in the raw (uncompressed) state.
 collects statuses of the`cephclusters.ceph.rook.io` resources
 from Openshift Data Foundation Stack.
 
-API Reference:
-  https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+The Kubernetes API https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
 
-* Location in archive: config/storage/<namespace>/<name>.json
+* Location of serviceaccounts in archive: config/storage/{namespace}/cephclusters/{name}.json
+* See: docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
 * Id in config: clusterconfig/ceph_cluster
 * Since versions:
-  * 4.12+
+- 4.8.49
+- 4.9.48
+- 4.10.31
+- 4.11.2
+- 4.12
```

Comment:
Nit: This should be indented.

Reply:
It seems that I completely forgot how the old docs where structured. LOL

Let me fix it.